libs/language-server/src/lib/ast/expressions/internal-value-representation.ts

```diff
@@ -14,7 +19,50 @@ import {
   isTransformDefinition,
   isValuetypeAssignment,
 } from '../generated/ast';
-import type { WrapperFactoryProvider } from '../wrappers';
+import { type WrapperFactoryProvider } from '../wrappers';
+
+abstract class JayveeError extends Error {
```

Comment:
I think we should call it `ErroneousValue` instead. But might be personal preference, opinions?